subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/ivyservice/resolveengine/excludes/factories/NormalizingExcludeFactoryTest.groovy

```diff
@@ -85,14 +85,14 @@ class NormalizingExcludeFactoryTest extends Specification {
 
         where:
         left                               | right         | expected
-        everything()                       | nothing()     | nothing()
-        everything()                       | everything()  | everything()
-        nothing()                          | nothing()     | nothing()
-        everything()                       | group("foo")  | group("foo")
-        nothing()                          | group("foo")  | nothing()
-        group("foo")                       | group("foo")  | group("foo")
+//        everything()                       | nothing()     | nothing()
```

Comment:
I believe these tests should remain active.

Reply:
oops of course